modules/test/playwright/tests/frontend-data-set-views-web/pages/ActionsPage.ts

```diff
@@ -49,9 +49,13 @@ export class ActionsPage {
 
 	async goto() {
 		await this.viewsPage.goto();
-		await this.viewsPage.gotoSampleDataSetView().then(() => {
-			this.page.getByRole('button', {name: 'Actions'}).first().click();
-		});
+		await this.viewsPage.gotoSampleDataSetView();
+
+		await this.page
+			.locator('li')
```

Comment:
Would `getByRole('listitem')` work?

Reply:
The `getByRole('listitem')`  has been failing randomly. This happened because the `Views` page also has an `Actions` button (dropdown to edit/delete), so sometimes the navigation is slow and it end up clicking in this button and get stuck there.
I've changed the selector trying to find the `Actions` section. 

![image](https://github.com/liferay-frontend/liferay-portal/assets/132653342/8e3744cd-b9fc-4a5d-8936-89b87f716521)